package.json

```diff
@@ -7,7 +7,7 @@
     "start": "webpack serve --mode=development --open",
     "build-web": "webpack",
     "build-parse": "tsc parsestart.ts --esModuleInterop --moduleResolution node --outDir cli/",
-    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' node --experimental-wasm-bigint node_modules/.bin/mocha -r ts-node/register --reporter mochawesome 'tests/**/*.test.ts'"
+    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' node --experimental-wasm-bigint node_modules/mocha/bin/mocha -r ts-node/register --reporter mochawesome 'tests/**/*.test.ts'"
```

Comment:
Why change the test command?

Reply:
Oops, sorry! The path for mocha is slightly different for me so I had to change this to make the tests work (I think it's because I'm on Windows?)

I'll make another commit to revert this!

Reply:
Hm if it works on *nix and it's necessary for windows, we could leave it. Really I just wanted to know. It works fine on my mac so let's leave the change.